Created Quickstart for OSS Portal Deployment and updated a couple other docs

CaitlinV39 opened the pull request:
## Description
Moving the quickstart guides to the OSS repo and adding the images for them. Also removed some duplicate text from Bulk Export that is covered in PaaS documentation so we don't have to keep it up to date in two places and added some links to the search page.

## Related issues
Addresses #1274

CaitlinV39 commented:
@brendankowitz or @LTA-Thinking wondering if you could take a look at my changes. Let me know if someone else should be added as a reviewer!

LTA-Thinking commented:
Why are we removing the documentation on connecting with Postman? I know we have a Microsoft docs page that covers that workflow, could we link to that?

LTA-Thinking commented:
In the default deployment doc we switch between saying a Cosmos DB or SQL instance will be deployed. The button on the page deploys SQL, but the command line instructions deploy Cosmos DB. I think it need to be clearer that there are different templates for the different data stores.

brendankowitz commented:
Looks good, I also have this random wiki page for getting started with dev, let me know if you wanted me to move this: https://github.com/microsoft/fhir-server/wiki/Build-and-debug-locally

brendankowitz commented:
> In the default deployment doc we switch between saying a Cosmos DB or SQL instance will be deployed. The button on the page deploys SQL, but the command line instructions deploy Cosmos DB. I think it need to be clearer that there are different templates for the different data stores.

I think our wording should be along the lines of we recommend starting with the sql deployment as it offers the most features. However deployments in cosmos are still available...

CaitlinV39 commented:
> > In the default deployment doc we switch between saying a Cosmos DB or SQL instance will be deployed. The button on the page deploys SQL, but the command line instructions deploy Cosmos DB. I think it need to be clearer that there are different templates for the different data stores.
> 
> I think our wording should be along the lines of we recommend starting with the sql deployment as it offers the most features. However deployments in cosmos are still available...

I updated this and also changed the powershell to reference the SQL template instead. LMK what you think

brendankowitz commented on docs/DefaultDeployment.md:
```diff
@@ -68,7 +70,7 @@ The template can also be deployed using PowerShell. Here is an example of how th
 
 New-AzResourceGroupDeployment `
--TemplateUri "https://raw.githubusercontent.com/Microsoft/fhir-server/master/samples/templates/default-azuredeploy.json" `
+-TemplateUri "https://raw.githubusercontent.com/Microsoft/fhir-server/master/samples/templates/default-azuredeploy-sql.json" `
```
I don't think this will just work as sql requires extra/different parameters, e.g. sql server password

LTA-Thinking replied:
I think the two that would be required would be sql server password and sql schema automatic updates enabled.

CaitlinV39 replied:
Thanks! I reverted and just made it clear that its Cosmos
